src/eval/str.rs

```diff
@@ -297,6 +297,11 @@ impl Str {
 
         Ok(resolved)
     }
+
+    /// Enumerate the grapheme clusters of this string.
+    pub fn enumerate(&self) -> Array {
+        self.clusters().enumerate()
+    }
```

Comment:
not sure about this one. I think an explicit `.clusters().enumerate()` in Typst code would be better since there are other things you might want to enumerate (e.g. codepoints)